routes/uploads.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Rename the file in storage.js and update the links to this file in order to make it more generic 😉